toolbox/_common.py

```diff
@@ -0,0 +1,97 @@
+import itertools
+import subprocess
+import os
+import time
+import sys
+from pathlib import Path
+
+top_dir = Path(__file__).resolve().parent.parent
+
+
+class PlaybookRun:
+    """
+    Playbook runner
+
+    !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
Wow, I thought 2 exclamation marks is too much. :)

Reply:
Yeah it's printed along with a bunch of other uninteresting information, so I wanted to make sure it grabs the user's attention before they waste their time on the uninteresting information.